2025/docs/4-insecure-authentication.md

```diff
@@ -1 +1,44 @@
-TBA
\ No newline at end of file
+# NHI4:2025 Insecure Authentication
+
+| Threat Agents & Attack Vectors                    | Security Weakness                                                                                          | Impact                                         |
+|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------|------------------------------------------------|
+| Exploitability: **Easy**            | Prevalence: **Widespread**<br>Detectability: **Hard**                       | Technical Impact: **Severe**     |
```

Comment:
Detectability is marked as "Easy" in the doc: https://docs.google.com/document/d/18Bu2ixzbxWFP-OBt7ZeOojBsOQ9daqUc8jw6uETbOO0/edit?tab=t.0#heading=h.mrv1n2cxy4m7

Reply:
Also, Technical Impact is marked as "Moderate" in the doc: https://docs.google.com/document/d/18Bu2ixzbxWFP-OBt7ZeOojBsOQ9daqUc8jw6uETbOO0/edit?tab=t.0#heading=h.mrv1n2cxy4m7

Reply:
Please also add "Business Impact: Specific" under "Impact"

Reply:
Don't know how i confused those, Thanks!
Opened another commit with the fixes 👍